projects/backend/src/main/java/backend/App.java

```diff
@@ -3,12 +3,33 @@
  */
 package backend;
 
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+import java.util.Arrays;
+
+@SpringBootApplication
 public class App {
-    public String getGreeting() {
-        return "Hello world.";
-    }
 
     public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+        SpringApplication.run(App.class, args);
+    }
+
+    @Bean
+    public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
+        return args -> {
+
+            System.out.println("Let's inspect the beans provided by Spring Boot:");
+
+            String[] beanNames = ctx.getBeanDefinitionNames();
+            Arrays.sort(beanNames);
+            for (String beanName : beanNames) {
+                System.out.println(beanName);
+            }
+
+        };
```

Comment:
Pewnie było użyte tylko w celu sprawdzenia czy projekt się kompiluje ale jest to niepotrzebne.
Wg mnie do usunięcia

Reply:
Przy dodawaniu nowy zależności do aplikacji może się ten moment przydać. Zgadzam się, że nie jest to niezbędne, ale na razie bym to zostawił.

Reply:
Może w takim razie przeniesc taka funkcjonalnosc do testów? 
W ten sposób upewnimy się że projekt się odpowiednio buduje i nie będziemy zaśmiecać  kodu produkcyjnego

Reply:
Dobra, narazie to wyleciało